bootstrapvz/common/tasks/grub.py

```diff
@@ -25,14 +25,53 @@ class ConfigureGrub(Task):
 	def run(cls, info):
 		from bootstrapvz.common.tools import sed_i
 		grub_def = os.path.join(info.root, 'etc/default/grub')
+
+		# optional grub settings
+		grub_settings = {}
+		if 'grub' in info.manifest.system:
+			grub_settings = info.manifest.system['grub']
+
+		# disable the graphical grub menu
 		sed_i(grub_def, '^#GRUB_TERMINAL=console', 'GRUB_TERMINAL=console')
-		sed_i(grub_def, '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
-		                'GRUB_CMDLINE_LINUX_DEFAULT="console=hvc0"')
-		sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=0\n'
-		                                        'GRUB_HIDDEN_TIMEOUT=0\n'
-		                                        'GRUB_HIDDEN_TIMEOUT_QUIET=true')
+
+		# configure the console
+		sed_i(
+		    grub_def,
+		    '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
+		    'GRUB_CMDLINE_LINUX_DEFAULT="console=hvc0 console=ttyS0"'
+		)
+
+		# disable persistent network interface names
+		from bootstrapvz.common.releases import jessie
+		if info.manifest.release > jessie:
```

Comment:
While virtually this is really the same as `info.manifest.release >= stretch`, semantically there is a difference. We specifically add something for stretch and higher up rather than doing something as long as its not jessie or lower. When reading the code you want to actually see the word `stretch`, this way you know what release some specific code concerns itself with.